distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1281,6 +1281,19 @@ A report is generated in _xls_ format.
 
 The ${branding} contains many tools to monitor system functionality, usage, and overall system health.
 
+===== Using the Log Viewer
+
+Monitor system logs with the `LogViewer`.
+
+* Navigate to the `LogViewer` at \${secure_url}/admin/logviewer
+
+Log events can be filtered by:
+
+* Log level (`ERROR`, `WARNING`, etc).
+** Logging may need to be configured to capture `DEBUG` OR `TRACE` messages.
+* Log message text.
+* Bundle generating the message.
```

Comment:
We should explain which messages are captured and displayed, and how many will be kept. We may also want to add a note or warning about setting the logging level to `TRACE` as that will flood the log viewer and make it less useful.

@garrettfreibott can provide more details.